code/game/gamemodes/cult/ritual.dm

```diff
@@ -177,6 +177,58 @@ var/global/list/rnwords = list("ire","ego","nahlizet","certum","veri","jatkaa","
 /obj/effect/rune/proc/check_icon()
 	icon = get_uristrune_cult(word1, word2, word3)
 
+/obj/item/weapon/book/tome/Initialize()
+    . = ..()
+    ritualknife = new /obj/item/weapon/melee/cultknife(src)

```

Comment:
Might be a good idea to make it so a cultist examining the tome can tell if there's a knife in it or not.

Reply:
Also need to delete this on Destroy().